projects/dev/src/app/shared/models/group.model.ts

```diff
@@ -1,20 +1,59 @@
-export interface Group {
+import { ManagementLevel } from '../constants/group';
+
+export class Group {
```

Comment:
Why not an interface as the other objects and not requiring conversion?

Reply:
Typescript interface isn't allowed to have member functions implementation.

Reply:
Right, it will be useful in the future at least.